src/cdk/table/table.ts

```diff
@@ -350,17 +379,24 @@ export class CdkTable<T> implements CollectionViewer {
     });
 
     this._changeDetectorRef.markForCheck();
+
+    return rowView;
+  }
+
+  /** Updates the row view ref with the implicit data assigned to the row. */
+  private _updateRowImplicitContext(view: RowViewRef<T>, record: IterableChangeRecord<any>) {
```

Comment:
I think the code would be easier to read without this method (just having the assignment inline). Any reason to keep it a method?